rosbag2_bag_v2_plugins/test/rosbag2_bag_v2_plugins/test_rosbag_v2_storage.cpp

```diff
@@ -76,7 +76,7 @@ TEST_F(RosbagV2StorageTestFixture, get_metadata_returns_bagfile_description)
 
   auto bag_metadata = storage_->get_metadata();
 
-  EXPECT_THAT(bag_metadata.version, Eq(1));
+  EXPECT_THAT(bag_metadata.version, Eq(2));
```

Comment:
```suggestion
  EXPECT_EQ(bag_metadata.version,2);
```

Reply:
'll open up an issue for replacing `EXPECT_THAT` with `EXPECT_EQ`